<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visitor</class>
 <widget class="QDialog" name="visitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLineEdit" name="relationLineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="visitorNameLineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>381</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 77, 56);
font: 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Visitor Info Panel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Enter Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Enter Relation:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="searchLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 77, 56);
font: 600 12pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Search Grave</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="graveTable">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>90</y>
       <width>391</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget {
    background-color: rgb(240, 255, 252); /* soft mint background for contrast */
    color: rgb(0, 50, 40); /* dark green text */
    gridline-color: rgb(180, 220, 210); /* soft line */
    border: 1px solid rgb(0, 112, 82);
    selection-background-color: rgb(0, 112, 82); /* selected row */
    selection-color: white;
    alternate-background-color: rgb(225, 245, 238); /* subtle alt row */
}

QTableWidget::item:hover {
    background-color: rgb(0, 112, 82);
    color: white;
}

QHeaderView::section {
    background-color: rgb(0, 112, 82);
    color: white;
    font-weight: bold;
    padding: 6px;
    border: 1px solid rgb(180, 220, 210);
}

QScrollBar:vertical, QScrollBar:horizontal {
    background: rgb(240, 255, 252);
    border: none;
    width: 12px;
    margin: 0px;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: rgb(0, 112, 82);
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical,
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none;
    border: none;
}

QTableWidget {
    background-color: rgb(240, 255, 252); /* soft mint background for contrast */
    color: rgb(0, 50, 40); /* dark green text */
    gridline-color: rgb(180, 220, 210); /* soft line */
    border: 1px solid rgb(0, 112, 82);
    selection-background-color: rgb(0, 112, 82); /* selected row */
    selection-color: white;
    alternate-background-color: rgb(225, 245, 238); /* subtle alt row */
}

QTableWidget::item:hover {
    background-color: rgb(0, 112, 82);
    color: white;
}

QHeaderView::section {
    background-color: rgb(0, 112, 82);
    color: white;
    font-weight: bold;
    padding: 6px;
    border: 1px solid rgb(180, 220, 210);
}

QScrollBar:vertical, QScrollBar:horizontal {
    background: rgb(240, 255, 252);
    border: none;
    width: 12px;
    margin: 0px;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: rgb(0, 112, 82);
    border-radius: 6px;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical,
QScrollBar::add-line:horizontal, QScrollBar::sub-line:horizontal {
    background: none;
    border: none;
}

QScrollBar:horizontal {
    background: rgb(240, 255, 252);
    border: none;
    height: 12px;  /* âœ… Corrected for horizontal */
    margin: 0px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cleanButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 77, 56);
color: rgb(255, 255, 255);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>clean grave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="waterButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 77, 56);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>give water</string>
     </property>
    </widget>
    <widget class="QPushButton" name="flowerButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 77, 56);
font: 12pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>add flower</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 112, 82);
color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
